meshtastic/tests/test_node.py

```
@@ -1247,6 +1302,226 @@ def test_setURL_add_only_raises_when_no_disabled_slot_available(
+    disabled = Channel(index=1, role=Channel.Role.DISABLED)
+    anode.channels = [primary, disabled]
+    before_snapshot = [channel.SerializeToString() for channel in anode.channels]
+    anode.localConfig.lora.hop_limit = 3
+    anode.writeChannel = MagicMock()  # type: ignore[method-assign]
+
+    failed_lora_send = {"seen": False}
+    rollback_messages: list[admin_pb2.AdminMessage] = []
+
+    def _send_admin_with_lora_failure(
+        msg: admin_pb2.AdminMessage,
+        wantResponse: bool = False,
```

Ruff (0.15.5)

[error] Boolean-typed positional argument in function definition

(FBT001)